src/genreclassutil.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import unicode_literals
+import re
+import subprocess
+import urllib.request
+import numpy as np
+import librosa
+import librosa.display
+import matplotlib.pyplot as plt
+import os
+from pydub import AudioSegment
+
+'''
```

Comment:
Just as a pointer, typical pep8 guidelines is to do a doc string under the function so like

```
def get_url(songIn, artistIn, index=0):
    """ Leverages urllib to send a search query to youtube.com 
    
    :param string songIn - the name of the song
    :param string artistIn - the name of the artist 
    :returns the full URL to a particular video for the given song
    """
```